services/pdf-ms/templateExample/ekb-löpande.json

```diff
@@ -0,0 +1,480 @@
+{
+  "numberOfpages": 5,
+  "defaultFontSize": 11,
+  "texts":
+  [
+  {
+      "page": 0,
+      "x": 400,
+      "y": 685,
+      "size": 11,
```

Comment:
size of what?

Reply:
The text. But I'll change it back to fontSize to be more explicit.